source/main/modules/window-manager/index.ts

```diff
@@ -219,6 +223,36 @@ export default class WindowManager {
       return
     }
 
+
+
```

Comment:
Fails `yarn lint` : `226:1   error    More than 1 blank line not allowed no-multiple-empty-lines`